docs/AboutUs.md

```diff
@@ -9,13 +9,12 @@ You can reach us at the email `seer[at]comp.nus.edu.sg`
 
 ## Project team
 
-### John Doe
+### Manas Bam
 
-<img src="images/johndoe.png" width="200px">
+<img src="images/manas-photo.png" width="200px">
```

Comment:
Will have to change the pathname to the updated one.
Also, would recommend shifting your info down to a different block, since I think this one is for our Tutor.